src/components/DefinitionEntry.js

```diff
@@ -261,71 +291,105 @@ export default class DefinitionEntry extends React.Component {
           </Row>
           <Row>
             <Col md={2}>{this.renderLabel('Release', true)}</Col>
-            <Col md={10}>
-              <InlineEditor
-                type="date"
-                initialValue={this.printDate(this.getOriginalValue('described.releaseDate'))}
-                value={this.printDate(this.getValue('described.releaseDate'))}
-                onChange={this.fieldChange('described.releaseDate')}
-                validator={value => true}
-                placeholder={'YYYY-MM-DD'}
-              />
+            <Col md={10} className="definition__line">
+              {this.renderWithToolTipIfDifferent(
+                'described.releaseDate',
+                <InlineEditor
+                  extraClass={this.classIfDifferent('described.releaseDate')}
+                  readOnly={readOnly}
+                  type="date"
+                  initialValue={this.printDate(this.getOriginalValue('described.releaseDate'))}
+                  value={this.printDate(this.getValue('described.releaseDate'))}
+                  onChange={this.fieldChange('described.releaseDate')}
+                  validator={value => true}
+                  placeholder={'YYYY-MM-DD'}
+                />
+              )}
             </Col>
           </Row>
           <Row>
             <Col md={2}>{this.renderLabel('Facets', true)}</Col>
-            <Col md={10}>
-              <p className="list-singleLine"> &nbsp;&nbsp;&nbsp;&nbsp;{this.printArray(initialFacets)}</p>
+            <Col md={10} className="definition__line">
+              {this.renderWithToolTipIfDifferent(
+                'described.facets',
+                <p className={`list-singleLine ${this.classIfDifferent('described.facets')}`}>
+                  {readOnly ? null : <span>&nbsp;&nbsp;&nbsp;&nbsp;</span>}
+                  {this.printArray(initialFacets)}
+                </p>
+              )}
             </Col>
           </Row>
         </Col>
         <Col md={7}>
           <Row>
             <Col md={2}>{this.renderLabel('Discovered')}</Col>
-            <Col md={10}>
-              <p className="list-singleLine">{get(licensed, 'discovered.expressions', []).join(', ')}</p>
+            <Col md={10} className="definition__line">
+              {this.renderWithToolTipIfDifferent(
+                'discovered.expressions',
+                <p className={`list-singleLine ${this.classIfDifferent('licensed.discovered.expressions')}`}>
+                  {get(licensed, 'discovered.expressions', []).join(', ')}
+                </p>
+              )}
             </Col>
           </Row>
           <Row>
             <Col md={2}>{this.renderLabel('Attribution', true)}</Col>
-            <Col md={10}>
-              <p className="list-singleLine">{get(licensed, 'attribution.parties', []).join(', ')}</p>
+            <Col md={10} className="definition__line">
+              {this.renderWithToolTipIfDifferent(
+                'attribution.parties',
```

Comment:
should this be `licensed.attribution.parties`? if not, why not? all the others match the prop name used in the next line for `classIfDifferent`.